src/Data/Integer/Properties.agda

```diff
@@ -1674,6 +1910,103 @@ m⊓n≤n (+ m)    (+ n)    = +≤+ (ℕₚ.m⊓n≤n m n)
 m⊓n≤m : ∀ m n → m ⊓ n ≤ m
 m⊓n≤m m n rewrite ⊓-comm m n = m⊓n≤n n m
 
+mono-≤-distrib-⊓ : ∀ f → f Preserves _≤_ ⟶ _≤_ → ∀ m n → f (m ⊓ n) ≡ f m ⊓ f n
+mono-≤-distrib-⊓ f f-mono-≤ m n with ≤-total m n
+... | inj₁ m≤n = trans (cong f (m≤n⇒m⊓n≡m m≤n)) (sym (m≤n⇒m⊓n≡m (f-mono-≤ m≤n)))
+... | inj₂ m≥n = trans (cong f (m≥n⇒m⊓n≡n m≥n)) (sym (m≥n⇒m⊓n≡n (f-mono-≤ m≥n)))
+
+mono-<-distrib-⊓ : ∀ f → f Preserves _<_ ⟶ _<_ → ∀ m n → f (m ⊓ n) ≡ f m ⊓ f n
+mono-<-distrib-⊓ f f-mono-< m n with <-cmp m n
+... | tri< m<n _    _   = trans (cong f (m≤n⇒m⊓n≡m (<⇒≤ m<n))) (sym (m≤n⇒m⊓n≡m (<⇒≤ (f-mono-< m<n))))
+... | tri≈ _   refl _   = trans (cong f (m≤n⇒m⊓n≡m ≤-refl)) (sym (m≤n⇒m⊓n≡m ≤-refl))
+... | tri> _   _    m>n = trans (cong f (m≥n⇒m⊓n≡n (<⇒≤ m>n))) (sym (m≥n⇒m⊓n≡n (<⇒≤ (f-mono-< m>n))))
+
+------------------------------------------------------------------------
+-- Other properties of _⊓_ and _*_
+
+*-distribˡ-⊓-nonNeg : ∀ m n o → + m * (n ⊓ o) ≡ (+ m * n) ⊓ (+ m * o)
+*-distribˡ-⊓-nonNeg zero    _ _ = refl
+*-distribˡ-⊓-nonNeg (suc m) = mono-≤-distrib-⊓ (+[1+ m ] *_) (*-monoˡ-≤-pos m)
+
+*-distribʳ-⊓-nonNeg : ∀ m n o → (n ⊓ o) * + m ≡ (n * + m) ⊓ (o * + m)
+*-distribʳ-⊓-nonNeg zero    n o = begin-equality
+  (n ⊓ o) * + zero                   ≡⟨ *-zeroʳ (n ⊓ o) ⟩
+  + zero                             ≡⟨⟩
+  + zero ⊓ + zero                    ≡˘⟨ cong₂ _⊓_ (*-zeroʳ n) (*-zeroʳ o) ⟩
+  (n * + zero) ⊓ (o * + zero)        ∎
+  where
+    open ≤-Reasoning
+*-distribʳ-⊓-nonNeg (suc m) = mono-≤-distrib-⊓ (_* +[1+ m ]) (*-monoʳ-≤-pos m)
+
+------------------------------------------------------------------------
+-- Structures
+
+⊓-isMagma : IsMagma _⊓_
+⊓-isMagma = Magma.isMagma (magma _⊓_)
```

Comment:
You can just use `isMagma _⊓_` rather than `Magma.isMagma (magma _⊓_)` where `isMagma` is from the same file as `magma`